tests/test_api_transfer_value.py

```diff
@@ -63,6 +63,27 @@ def test_transfer_case0(self):
         except FileNotFoundError:
             self.assertFalse(True)
 
+    def test_transfer_case0_0(self):
+        """Test for transfer_value_with_the_fee function.
+         Case when transfer 0.1(float) value to a wallet.
+        """
+
+        # Given
+        password = "ejfnvm1234*"
+        file_path = os.path.join(TEST_DIR, "test_keystore_for_transfer.txt")
+
+        # When
+        try:
+            ret = bool(wallet.transfer_value_with_the_fee(
```

Comment:
```
 ret = bool(wallet.transfer_value_with_the_fee(
                password = password, amount = 10000000000000000, to="hxa974f512a510299b53c55535c105ed962fd01ee2",
                amount="0.5", file_path=file_path, url=url))
```

Ane use `e` to express long integer.  Here is the example. 

```
>>> value1 = 10000000000000000
>>> value2 = 1.e16
>>> value2.is_ineteger()
True 
>>> value1==value2
True
```
I think it's better to read the code. ;-)